tests/HighlighterTest.php

```diff
@@ -41,4 +42,16 @@ public static function data(): array
             ['02', 'html'], // deep injections
         ];
     }
+
+    public function test_has_language(): void
+    {
+        $highlight = new Highlighter();
+
+        Assert::assertTrue($highlight->hasLanguage('php'));
```

Comment:
For about 10 years, from PHPUnit 1.0 in 2006 until PHPUnit 5.0 in Oct 2015, PHPUnit needed to use `$this->assert()` methods.

However, with the adoption of PHP 5.6 and the new `static` mechanism, it offloaded all of the assert functions to a standalone class `PHPUnit\Framework\Assert`, which later (in the PHPUnit 9 days) became its own packagist repo in its own right.

Calling static methods as regular instance methods is considered a bad practice. Because of this, with the release of PHPUnit 7.0, in 2019, there was [a discussion](https://github.com/sebastianbergmann/phpunit/issues/1914#issuecomment-471978001) about whether one should still use `$this->` or `self::` for the methods. 

Two camps emerged: If you use static analysis, believe in strong typing, then prefer `self::`. If you prefer the dynamic nature and loose typing of PHP in a by-gone-tho-still-supported era, use `$this->`.

Because the `php-cs-fixer` of this project enforces `$this->`, I've decided to just call the `Assert` class directly. It's faster and directly equivalent to `self::`, so it's actually even better, from a technical standpoint.

Reply:
@hopeseekr - This has been addressed [here](https://github.com/tempestphp/highlight/pull/63#discussion_r1545372313) and the stance of this project will not change based on your assertions of best practice, particularly on such a trivial thing. The `php-cs-fixer` represents the style choices made and exists for a reason.

Contributions are welcomed and valued, but pushing your own opinions upon the project is not helpful to forward progress.

Reply:
I apologize and will not repeat the mistake.